nautobot/docs/installation/http-server.md

```diff
@@ -138,16 +138,47 @@ Finally, restart the `nginx` service to use the new configuration.
 $ sudo systemctl restart nginx
 ```
 
+!!! info
+    If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS 8.2.2 requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
+
+## Confirm Permissions for /opt/nautobot
+
+Ensure that the `/opt/nautobot/` permissions are set to `755`.
+
+Example of correct permissions:
+```no-highlight
+[root@localhost ~]# ls -l /opt/
+total 4
+drwxr-xr-x. 11 nautobot nautobot 4096 Apr  5 11:24 nautobot
+[root@localhost ~]# 
+```
+
+If the permissions are not correct, modify them accordingly.
+
+Example of modifying the permissions:
+```no-highlight
+[nautobot@localhost ~]$ ls -l /opt/
+total 4
+drwx------. 11 nautobot nautobot 4096 Apr  5 10:00 nautobot
+[nautobot@localhost ~]$ chmod 755 /opt/nautobot/
+[nautobot@localhost ~]$ ls -l /opt/
+total 4
+drwxr-xr-x. 11 nautobot nautobot 4096 Apr  5 11:24 nautobot
+```
```

Comment:
It's not clear what needs to be done without following this whole section. IMO just replace this with a brief admonition under the system user creation that says "On CentOS run `sudo chmod 755 /opt/nautobot`".

I do like having the verbose steps with command output, but I would rather see that in the "troubleshooting" section towards the bottom vs. in the deployment instructions.

Reply:
Roger that.  I'm not certain if changing the permissions on that directory as `root` would muck with the ownership, so I just did this for the time being as I didn't have time to go back and validate it:

```
## Confirm Permissions for /opt/nautobot

Ensure that the `/opt/nautobot/` permissions are set to `755`.
If permissions need to be changed, as the `nautobot` user run: `$ chmod 755 /opt/nautobot/`
```

I added the verbose troubleshooting in the troubleshooting section towards the bottom.

Reply:
I will go thru the permissions thing Thursday morning and revamp if possible to what you suggest